src/components/Guilds/Sidebar/SidebarMobile.tsx

```diff
@@ -0,0 +1,152 @@
+import styled from 'styled-components';
+import { Box } from '../common/Layout';
+import { Button } from '../common/Button';
+import { Menu, MenuItem } from '../common/Menu';
+import dxIcon from '../../../assets/images/dxdao-icon.svg';
+import { useState } from 'react';
+
+const SidebarHeader = styled(Box)`
+  border-bottom: 1px solid #000;
+  padding: 1rem 0rem;
+`;
+
+const DaoInfoPanel = styled(Box)`
+  padding: '1rem';
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+`;
+const DaoInfoRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+const DaoBrand = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+/*
+const DaoDetails = styled(Box)<{ shrink: boolean }>`
+  display: flex;
+  flex-direction: column;
+  align-items: ${props => (props.shrink ? 'center' : 'flex-start')};
+  padding-left: ${props => (props.shrink ? '0' : '1rem')};
+`;
+*/
+const UserInfoPanel = styled(Box)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1.2rem;
+  border-top: 1px solid #000;
+`;
+
+const UserInfoDetail = styled(Box)`
```

Comment:
so, here SidebarMobile and DesktopMobile share some common styles (the ones that are the same for mobile and desktop), should we separate them in a common.styled.ts file ?